src/sinks/util/service.rs

```diff
@@ -97,7 +97,7 @@ pub struct TowerRequestConfig {
 
     /// The time a request can take before being aborted.
     ///
-    /// It is highly recommended that you do not lower this value below the service’s internal timeout, as this could
+    /// Datadog highly recommends that you do not lower this value below the service's internal timeout, as this could
```

Comment:
@maycmlee should this say Datadog here since it's shared between the DD docs and Vector?

Reply:
Just wanted to confirm it was intentional, just caught me by surprise when I saw Datadog on all of the sinks